outputs.tf

```diff
@@ -0,0 +1,14 @@
+output "domain_arn" {
+  value       = aws_elasticsearch_domain.this.arn
+  description = "string ||| ARN of the OpenSearch domain"
+}
+
+output "domain_endpoint" {
```

Comment:
There is an implicit convention I've been following with datastores for outputs that makes it easier to hook up capabilities.

Notably, I've started ensuring that the outputs include `db_protocol`, `db_endpoint`, `db_security_group_id`.
- `db_protocol` would be `https`.